css/elements.css

```
@@ -78,16 +78,35 @@ emu-const {
 emu-val {
   font-weight: bold;
 }
-emu-alg ol, emu-alg ol ol ol ol {
-    list-style-type: decimal;
-}
-
-emu-alg ol ol, emu-alg ol ol ol ol ol {
-    list-style-type: lower-alpha;
-}
 
-emu-alg ol ol ol, ol ol ol ol ol ol {
-    list-style-type: lower-roman;
+emu-alg ol,
+emu-alg ol ol ol ol,
+emu-alg ol.nested-thrice,
+emu-alg ol.nested-twice ol,
+emu-alg ol.nested-once ol ol {
+  list-style-type: decimal;
+}
+
+emu-alg ol ol,
+emu-alg ol ol ol ol ol,
+emu-alg ol.nested-once,
+emu-alg ol.nested-thrice ol,
+emu-alg ol.nested-twice ol ol,
+emu-alg ol.nested-four-times,
+emu-alg ol.nested-once ol ol ol {
+  list-style-type: lower-alpha;
+}
+
+emu-alg ol ol ol,
+emu-alg ol ol ol ol ol ol,
+emu-alg ol.nested-twice,
+emu-alg ol.nested-once ol,
+emu-alg ol.nested-four-times ol,
+emu-alg ol.nested-thrice ol ol,
+emu-alg ol.nested-twice ol ol ol,
+emu-alg ol.nested-once ol ol ol ol,
+emu-alg ol.nested-lots {
```

```suggestion
emu-alg ol ol ol,
emu-alg ol.nested-twice,
emu-alg ol.nested-once ol,
emu-alg ol ol ol ol ol ol,
emu-alg ol.nested-lots,
emu-alg ol.nested-four-times ol,
emu-alg ol.nested-thrice ol ol,
emu-alg ol.nested-twice ol ol ol,
emu-alg ol.nested-once ol ol ol ol,
emu-alg ol.nested-lots ol {
```

Did some re-ordering and also I think you missed `emu-alg ol.nested-lots ol`.

---

> `emu-alg ol.nested-lots ol`

That shouldn't be necessary; it's implied by `ol.nested-lots`. No?

---

According to my reading, the inner `<ol>` of `<emu-alg><ol class="nested-lots><ol> ...` would only be matched by `emu-alg ol ol`, which would give it `list-style-type: lower-alpha` not `list-style-type: lower-roman`.

---

Yup. Updated.